imports/plugins/included/product-detail-simple/client/components/variantList.js

```diff
@@ -148,7 +166,9 @@ VariantList.propTypes = {
   childVariants: PropTypes.arrayOf(PropTypes.object),
   displayPrice: PropTypes.func,
   editable: PropTypes.bool,
+  isDigital: PropTypes.any,
```

Comment:
Yes However I had to initialize the state as an empty string and Its also one of the conditions on line 131